src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/PkcsFormatReader.cs

```diff
@@ -218,29 +218,29 @@ internal static bool TryReadPkcs7Pem(SafeBioHandle bio, out List<ICertificatePal
             return true;
         }
 
-        internal static bool TryReadPkcs12(byte[] rawData, string password, out ICertificatePal certPal)
+        internal static bool TryReadPkcs12(byte[] rawData, object password, out ICertificatePal certPal)
```

Comment:
Let's take offline. There's one one marshal at the point of use. Breaking it early not ideal either as it exposes the string longer and in some code paths may not be necessary. SafeHandle ideal, but perhaps overkill unless we have another use for it.